app/app_test.go

```diff
@@ -38,7 +38,7 @@ func TestNew_PristineJoiner(t *testing.T) {
 
 // Open a database on a fresh one-node cluster.
 func TestOpen(t *testing.T) {
-	app, cleanup := newApp(t)
+	app, cleanup := newApp(t, app.WithAddress("127.0.0.1:9001"))
```

Comment:
If the other comments get addressed, this change could be reverted as well, so we test with the default which is to pick the first non-loopback address on the host.